File: Data Structures/anagrams.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;
public class Solution {
    public static int numberNeeded(String first, String second) {
        HashMap<String,Integer> alphacount = new HashMap<>();
        for (char ch = 'a'; ch <= 'z'; ++ch){
            alphacount.put(String.valueOf(ch), 0);
        }
        for (int i = 0; i < first.length(); i++){
            alphacount.put(Character.toString(first.charAt(i)), alphacount.get(Character.toString(first.charAt(i)))+1);
        } // array of numbers of occurences of each letters
        for (int i = 0; i < second.length(); i++){
            alphacount.put(Character.toString(second.charAt(i)), alphacount.get(Character.toString(second.charAt(i)))-1);
        }
        int diff = 0;
        for (String key : alphacount.keySet()){
            diff = diff + Math.abs(alphacount.get(key));
        }
        return diff;
    }
  
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String a = in.next();
        String b = in.next();
        System.out.println(numberNeeded(a, b));
    }
}

File: Data Structures/anagrams.py
def number_needed(a, b):
    import string
    alphacount = dict.fromkeys(string.ascii_lowercase, 0);
    for letter in a:
        alphacount[letter] = alphacount[letter]+1
    for letter in b:
        alphacount[letter] = alphacount[letter]-1
    diff = 0
    for key, value in alphacount.items():
        diff = diff + abs(value)
    return diff    
        

a = input().strip()
b = input().strip()

print(number_needed(a, b))
